### ISSUE TITLE
Error doesn't occur when npm installing was failed

### ISSUE THREAD

**Issue by anseki** [hakobera/nvmw#38]
The npm installing was failed (network problem, maybe), but any error doesn't occur, and a version without npm was set to using.
I think that an error should occur, but this behavior has some reason?

https://github.com/hakobera/nvmw/blob/master/nvmw.bat#L154
```shell
  if not exist %NPM_ZIP_FILE% (
    exit /b 0;
  )
```

```shell
  if not exist "%NPM_ZIP_FILE%" goto install_error
```

(And, I think that that path should be quoted.)

**Comment by hakobera** [comment]
@anseki There are no reason to that behavior. It's my mistake.
This issue is solved by #39? (Yes, it's your pull request)

**Comment by anseki** [comment]
Yes, This issue is solved. :smile: Thank you!
